device/bgp/bgpsrteadvanced.yaml

```diff
@@ -0,0 +1,25 @@
+components:
+  schemas:
+    BgpSrTe.Advanced:
+      description: >-
+        Configuration for advanced BGP route range settings.
+      type: object
+      properties:
+        origin:
+          description: >-
+            The origin value. 
+            A null value means the origin feature is not enabled.
+          type: string
+          enum:
+          - igp
+          - egp
```

Comment:
Probably  'incomplete' should be the third possible value of 'origin'

Reply:
Its is missing in regular Route Range(RR). We can add this in both RR and SRTE together in separate PR which will be a backward compatible change.